packages/target-decisioning-engine/src/filters.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+import { isUndefined } from "@adobe/target-tools";
+
+export function byPropertyToken(propertyToken) {
+  /**
+   * @param {import("../types/DecisioningArtifact").Rule} rule
+   * @return {boolean}
+   */
+  function filter(rule) {
+    const { propertyTokens = [] } = rule;
+
+    return isUndefined(propertyToken)
+      ? propertyTokens.length === 0
```

Comment:
```suggestion
      ? !propertyTokens.length
```

Reply:
I prefer `propertyTokens.length === 0` because I think it is more readable than `!propertyTokens.length`